src/Directory.Build.props

```diff
@@ -2,9 +2,9 @@
 
   <!-- TODO: (Consolidation) Remove when consolidated. -->
   <PropertyGroup>
-    <IsRuntimeRepository Condition="Exists('..\..\.dotnet-runtime-placeholder')">true</IsRuntimeRepository>
+    <IsRuntimeRepository Condition="Exists('$(MSBuildThisFileDirectory)..\..\.dotnet-runtime-placeholder')">true</IsRuntimeRepository>
```

Comment:
Are you sure this change alone isn't the fix? It would make sense to me if the condition is evaluated in context of the importing project.